src/arch/host/lib/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+add_local_sources(sof notifier.c)
```

Comment:
You can add newline if you are going to do more changes anyway. There is no need to retrigger Ci if everything else is ok.

Reply:
done